build/simp-utils.spec

```diff
@@ -1,7 +1,50 @@
+%{lua:
+--
+-- When buildng the RPM, declare macro 'pup_module_info_dir' with the path to
```

Comment:
See comments in https://github.com/simp/rubygem-simp-cli/pull/169#discussion_r683434314